src/main/java/com/fincatto/documentofiscal/utils/DFKeyManager.java

```diff
@@ -0,0 +1,80 @@
+package com.fincatto.documentofiscal.utils;
+
+import com.fincatto.documentofiscal.DFConfig;
+
+import javax.net.ssl.X509KeyManager;
+import java.net.Socket;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.Principal;
+import java.security.PrivateKey;
+import java.security.cert.Certificate;
+import java.security.cert.X509Certificate;
+import java.util.Enumeration;
+
+/**
+ * Created by Eldevan Nery Junior on 18/10/19.
+ *
+ */
+public class DFKeyManager implements X509KeyManager {
+
+    private KeyStore ks;
+    private String alias;
+    private String password;
+
+    public DFKeyManager(DFConfig config) throws KeyStoreException {
+        this.ks = config.getCertificadoKeyStore();
+        this.alias = getAlias(this.ks);
+        this.password = config.getCertificadoSenha();
+    }
+
+    public String chooseClientAlias(String[] str, Principal[] principal, Socket socket) {
+        return alias;
+    }
+
+    public String chooseServerAlias(String str, Principal[] principal, Socket socket) {
+        return alias;
+    }
+
+    public String[] getClientAliases(String str, Principal[] principal) {
+        return new String[]{alias};
+    }
+
+    public String[] getServerAliases(String str, Principal[] principal) {
+        return new String[]{alias};
+    }
+
+    public X509Certificate[] getCertificateChain(String alias) {
+        try {
+            Certificate[] certificates = this.ks.getCertificateChain(alias);
+            X509Certificate[] x509Certificates = new X509Certificate[certificates.length];
+            System.arraycopy(certificates, 0, x509Certificates, 0, certificates.length);
+            return x509Certificates;
+        } catch (KeyStoreException e) {
+            System.out.println("Não foi possível carregar o keystore para o alias:" + alias);
```

Comment:
Utilizar a lib de log, nao sysout, para permitir que quem for usar a lib direcione o log para o local correto.